pkg/adapter/quotas.go

```diff
@@ -36,28 +36,38 @@ type (
 		Builder
 
 		// NewQuotasAspect returns a new instance of the Quota aspect.
-		NewQuotasAspect(env Env, c AspectConfig, d map[string]*QuotaDefinition) (QuotasAspect, error)
+		NewQuotasAspect(env Env, c AspectConfig, quotas map[string]*QuotaDefinition) (QuotasAspect, error)
 	}
 
-	// QuotaKind determines the usage semantics
-	QuotaKind int8
-
-	// QuotaDefinition is used to describe an individual quota the aspect will encounter at runtime
+	// QuotaDefinition is used to describe an individual quota that the aspect will encounter at runtime.
 	QuotaDefinition struct {
-		// MaxAmount specifies the upper limit for the quota
+		// Name of this quota definition.
+		Name string
+
+		// Optional user-friendly name for this quota.
```

Comment:
nit: go convention is to have comments start with the name of the thing being commented on and to be complete sentences. here, and below, this would change to comments like `// DisplayName is an optional user-friendly name for a quota definition`.